apps/docs/content/guides/auth/phone-login/textlocal.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: 'auth-textlocal'
+title: 'Phone Auth with Textlocal'
+description: 'How to set up and use Mobile OTP with Textlocal and Supabase.'
+---
+
+<Admonition type="tip">
+
+To manage costs related to SMS authentication, consider setting project rate limits and configuring CAPTCHA. For additional cost-control strategies and to ensure compliance with TRAI's DLT requirements, visit the Production Checklist.
```

Comment:
Pretty sure we don't have anything about TRAI DLT in the Production Checklist.

Also, is this an India-specific requirement? If so, maybe we could make that more clear, to avoid confusing users from other countries and make it more globally applicable.

```
Also check country-specific regulations around sending SMS messages, for example, India's TRAI DLT regulations.
```